src/_reducers/assetStatus.reducer.js

```diff
@@ -0,0 +1,46 @@
+import constants from '../_constants';
+import initialState from './initialState';
+
+const {
+  ASSET_STATUS_LOADING,
+  ASSET_STATUS_FETCH_SUCCESS,
+  ASSET_STATUS_FETCH_FAILURE
+} = constants;
+
+export default(state = initialState.assetStatus, action) => {
```

Comment:
Shouldn't there be a space between `default` and `(`?  That is,
```js
export default (state = initialState.assetStatus, action) => {
```
You can verify this in the [MDN docs](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#Description)